.github/workflows/buildMLTBXTags.yml

```diff
@@ -24,10 +25,9 @@ jobs:
       
       - name: Create MLTBX
         uses: matlab-actions/run-command@v1
-        continue-on-error: true
+        continue-on-error: false
         if: always()
         with:
-          # command: matlab.addons.toolbox.packageToolbox("fsdaToolboxPackaging.prj", "release/fsda.mltbx")
```

Comment:
Comment for line 44: Are we not creating a draft release as Rob recommended?

Reply:
marked draft as true